tests/pytests/unit/modules/test_salt_version.py

```diff
@@ -64,6 +64,13 @@ def test_get_release_number_success_new_version():
     assert salt_version.get_release_number("Neon") == "3000"
 
 
+def test_get_release_number_success_new_version_with_dot():
+    """
+    Test that a version is returned for new versioning (3006)
+    """
+    assert salt_version.get_release_number("Sulfur") == "3006"
```

Comment:
```suggestion
    assert salt_version.get_release_number("Sulfur") == "3006.0"
```
In theory, this should be the expected outcome... Although, all 3006 minor versions are Sulfur versions...